rfc/manualjudgment.md

```diff
@@ -0,0 +1,68 @@
+# Select roles that can execute manual judgement stage
+
+| | |
+|-|-|
+| **Status**     | _**Proposed**, Accepted, Implemented, Obsolete_ |
+| **RFC #**      | https://github.com/spinnaker/deck/pull/7636
+| **Author(s)**  | Sanjeev Thatiparthi (@sanopsmx) (https://github.com/sanopsmx)
+| **SIG / WG**   | sig-ux
+
+## Overview
+
+Display all the roles the user have in the manual judgment stage.
```

Comment:
Could we get a  bit more detail here? Some questions I'd like to see answered:
1. Why are we displaying the roles?
2. What's the user/business story behind this feature? What kind of user would benefit from this?
3. Is this meant as a security or compliance feature?

I realize some of this is probably covered in the original issue but I'd like to see the details moved over here for visibility.